tests/lib/spread/rules/nightly.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
I don't see where we use this rules file

Reply:
It's used in the spread-nightly job

Reply:
but in nightly jobs you won't use this because this is not a PR, so it is going to execute all the tasks defined in tasks input.

Reply:
In any nightly job, the condition [ -z "${{ github.event.number }}" ] will match